grafana-stack/nginx.yaml

```diff
@@ -40,6 +40,7 @@ spec:
     targetPort: nginx-port
   selector:
     app: nginx
+  loadBalancerIP: 104.197.200.129
```

Comment:
Well, the service will need an IP when it's created. If `loadBalancerIP` is not specified, it will get a random IP. If you want it to use a specific IP (the one mapped by velodrome.k8s.io), it needs to know it